ui/src/dashboards/containers/DashboardPage.js

```diff
@@ -311,6 +284,28 @@ class DashboardPage extends Component {
     dashboardActions.putDashboardByID(dashboardID)
   }
 
+  handleSaveTemplateVariables = async templates => {
+    const {location, dashboardActions, dashboard} = this.props
+
+    try {
+      await dashboardActions.putDashboard({
```

Comment:
i don't believe either `putDashboard` or `syncURLQueryFromTempVars` throw errors, so i'm not sure if this `try/catch` is doing anything – thoughts? also, at least `putDashboard` should dispatch the error it catches so that the user would be shown a notification, which i would have otherwise requested here.